packages/trader/src/App/Components/Elements/PositionsDrawer/positions-modal-card.tsx

```diff
@@ -102,151 +93,15 @@ const PositionsModalCard = observer(
         const fallback_result = profit_loss >= 0 ? 'won' : 'lost';
         const total_profit = getTotalProfit(contract_info);
 
-        const should_show_sell = hasContractEntered(contract_info) && isOpen(contract_info);
         const display_name = getSymbolDisplayName(
             active_symbols,
             getMarketInformation(contract_info.shortcode || '').underlying
         );
 
-        const contract_options_el = (
```

Comment:
Instead of this old card, for all trade types (except `Vanillas`) will be shown `custom_contract_el`.

Reply:
contract_vanilla_el is for vanillas.
custom_contract_el is common for the rest.
@kate-deriv what do you think about renaming custom_contract_el to common_contract_el ? :)

Reply:
Will do